packages/create-pantheon-decoupled-kit/src/templates/next-drupal/.env.example

```diff
@@ -1,14 +1,23 @@
-# Copy as .env.development.local to override envars for local development
```

Comment:
Could you elaborate on the reasoning behind removing this? I feel like guidance like this is helpful and probably should be in the other example env files.

Reply:
No good reason, I will add it back.